ui.apps/src/main/content/jcr_root/apps/asset-share-commons/components/modals/license/license.html

```diff
@@ -39,10 +39,16 @@
             <img src="${asset.properties['thumbnail']}"/>
         </div>
         <div class="description cmp-content__iframe--wrapper">
-            <iframe data-sly-test="${asset.properties['license']}"
+            
+            <iframe data-sly-test="${wcmmode.edit && asset.properties['license']}"
```

Comment:
I would only check for wcmmode.disabled, as all other modes may switch on client side.